pkg/cmd/clients/factory.go

```diff
@@ -773,3 +775,8 @@ func (f *factory) CreateCertManagerClient() (certmngclient.Interface, error) {
 	}
 	return certmngclient.NewForConfig(config)
 }
+
+// NewKustomize creates a new Kustomize client
+func (f *factory) NewKustomize() kustomize.Kustomizer {
```

Comment:
I think to stick with the other functions in here, it should be `func (f *factory) CreateKustomizer`